Library/Homebrew/brew.sh

```diff
@@ -12,7 +12,19 @@ then
   then
     export LC_ALL="en_US.UTF-8"
   else
-    export LC_ALL="C.UTF-8"
+    utf8_locales=$(locale -a | grep --extended-regexp '\.(utf|UTF-)8')
+    re1='\bC\.(utf|UTF-)8'
```

Comment:
`c_regex`

Reply:
Why `\b`?

Reply:
word boundary (on Linux platforms) to avoid matching things like `xx_XC.`. I don't know if there are any locales that match that but thought it wouldn't hurt to to be on a safe side. Should I remove it?

Reply:
Fine with me 👍🏻